ios/tools/prepRelease.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 ## Note - this script is designed solely for use within our Continuous
-## Integration processes with `pwd` as this script's parent folder.
+## Integration processes (thus, by ../ci.sh) with `pwd` as this script's parent folder.
 
 # Exit if program call fails
 set -e
```

Comment:
I think this is handled by build-utils.sh (ll. 6-10 can be removed)

Reply:
For l.19 could it be
cd "$KEYMAN_ROOT/ios"

Reply:
My preference would be to rework this script's contents as inlined-code within ci.sh for the `prep-release` action.

Reply:
Yes, one step at a time is good. Moving the general build to builder format is a good incremental improvement.